pkg/omaha/omaha.go

```diff
@@ -69,7 +70,7 @@ func (h *Handler) Handle(rawReq io.Reader, respWriter io.Writer, ip string) erro
 
 	if err := xml.NewDecoder(rawReq).Decode(&omahaReq); err != nil {
 		logger.Warn("Handle - malformed omaha request", "error", err.Error())
-		return ErrMalformedRequest
+		return fmt.Errorf("%s: %s", ErrMalformedRequest, err.Error())
```

Comment:
Please use `%w` verb for errors. With it, `fmt.Errorf` will wrap `err`, which you can later retrieve. For more about it, see https://golang.org/doc/go1.13#error_wrapping
```suggestion
		return fmt.Errorf("%s: %w", ErrMalformedRequest, err)
```